src/Table2Beta/Table.tsx

```diff
@@ -472,10 +472,18 @@ export class Table2Beta extends React.Component<Props, State> {
           value={
             <>
               {singleActions[0].icon && (
-                <img className={cssClass.ACTION_ICON} src={singleActions[0].icon} />
+                <img
+                  className={cssClass.ACTION_ICON}
+                  src={singleActions[0].icon}
+                  alt={singleActions[0].title.singular.toString()}
+                />
               )}
               {singleActions[0].hoverIcon && (
-                <img className={cssClass.ACTION_HOVER_ICON} src={singleActions[0].hoverIcon} />
+                <img
+                  className={cssClass.ACTION_HOVER_ICON}
+                  src={singleActions[0].hoverIcon}
+                  alt={singleActions[0].title.singular.toString()}
+                />
               )}
               <div className={cssClass.ACTION_TITLE}>{singleActions[0].title.singular}</div>
             </>
```

Comment:
With this exact code block (I think) repeated below, can you capture the JSX in a variable and use that variable instead of duplicating? Especially with this markup appearing within a prop, `value`, it feels like it could simplify reading the `Button` component.

Reply:
Love it!